resources/public/api/conf/1.0/application.yaml

```diff
@@ -17,27 +26,31 @@ servers:
 tags:
   - name: Trigger monthly reconciliation report
     description: |
-      To trigger a reconciliation report send a request to the reconciliation endpoint outlining how many errors you want to recieve of each category
+      Simulate a sandbox callback from HMRC indicating that reconciliation results are available to retrieve through the ISA Returns API.
 
 #      Open Question - since we can reasonably assume they might want to be more specific, what does that endpoint look like?
 
 paths:
-  /monthly/{zRef}/{taxYear}/{month}/reconciliation:
+  /monthly/{zReference}/{taxYear}/{month}/reconciliation:
     post:
-      summary: Submit reconciliation endpoint
+      summary: Simulate reconciliation callback
       description: |
-        Use this endpoint to trigger the generation of a report matching the submitted conditions.
-      operationId: submitReconciliation
+        Use this endpoint to simulate a callback from HMRC indicating that reconciliation results are available to retrieve through the ISA Returns API.
+        
+        You can specify how many test errors to include in each category: oversubscriptions, trace and match failures, and eligibility failures. 
+        
+        The results will be returned to your software using the same callback mechanism used in the ISA Returns API.
```

Comment:
Can we add more detail to this endpoint description?

Reply:
https://github.com/hmrc/disa-returns-test-support-api/blob/32b10c756f405249b0694b6fbd962daeaf7f5772/resources/public/api/conf/1.0/application.yaml#L42C1-L43C1

I'm not sure I understand this description. What callback mechanism are you referring to? is this the PPNS notification that is sent via PPNS to the software via call back url that they can provide?